smac/scenario/scenario.py

```diff
@@ -18,6 +18,14 @@
 __version__ = "0.0.2"
 
 
+def _is_truthy(arg):
+    return arg in ["1", "true", True]
+
+def _parse_initial_incumbent(arg):
+    arg = str.upper(arg.strip())
+    return arg if arg in ['DEFAULT', 'RANDOM'] else None
```

Comment:
I think it should be possible. @Ayaro could you please try this and submit an update to this PR?